src/settings-ui/Settings.UI.Library/MouseJumpSettings.cs

```diff
@@ -7,6 +7,9 @@
 using System.Text.Json.Serialization;
 
 using Microsoft.PowerToys.Settings.UI.Library.Interfaces;
+using MouseJump.Common.Helpers;
+using MouseJump.Common.Models.Settings;
+using Windows.UI.Xaml.Printing;
```

Comment:
`Windows.UI.Xaml.Printing` doesn't seem to be used anywhere, but it's giving an analyzer error. Removing it.